pkg/steps/release/create_release.go

```diff
@@ -175,6 +176,21 @@ func (s *assembleReleaseStep) run(ctx context.Context) error {
 		return results.ForReason("missing_release").WithError(err).Errorf("could not resolve imagestream %s: %v", streamName, err)
 	}
 
+	// we want to expose the release payload as a CI version that looks just like
+	// the release versions for nightlies and CI release candidates
+	prefix := "0.0.1-0"
```

Comment:
So, now the prefix would be 0.0.1-0 instead of 4.<something> ?

Reply:
Only for the cases where we cannot determine the version, it's a fallback.

Reply:
oh, somehow  I missed that. Thanks!